hack/load-tests/log-fluentbit-test-setup.yaml

```diff
@@ -99,6 +99,7 @@ spec:
           name: flog
           resources:
             limits:
+              cpu: 50m
```

Comment:
In the metric performance tests the load generators are also missing CPU limits? Where they removed as well? Then please revert them as well.